Standoff

titannote opened the pull request.

titannote committed:
Standoff

titannote committed:
Merge branch 'development' into #108-Standoff

Velsh-arc committed:
Push fixed by Velsheda

Empty push is fixed, map works properly, global lightmap is disabled at the moment, clear hierarchy and tree fixes are needed

Lithrun reviewed (changes requested):
Map is still WIP. On Hold until actual PR can start

dark3art committed:
fixed some prefabs

dark3art committed:
fixed more prefabs

dark3art committed:
fixed the last prefabs

dark3art committed:
deleted wrong water shaders

Proidott committed:
test push

Proidott committed:
deleted multiplying buildings

some of the building's had multiplied specifically the watch towers

Proidott committed:
Made folders

ive put some stuff in folders for better organization its not everything but its still a lot

Proidott committed:
Folders

most the forest is now in one folder

Proidott committed:
everything is in folders

Proidott committed:
Placed more bulidings

ive placed houses along the paths to make a environment

Proidott committed:
added in more buildings

Proidott committed:
more buildings

all i have to do is resize them

Proidott committed:
Resized all the buildings

titannote committed:
Merge branch 'development' into #108-Standoff

titannote committed:
Water Shaders removed

titannote committed:
Added River

Proidott committed:
other side of river

Ive placed all the buildings on the other side of the river and have reworked the paths so they would look good

titannote committed:
Added Day Night Cycle

dark3art committed:
fixed missing terrain

dark3art committed:
added new watchtower models

dark3art committed:
fixed watchtowers maps

titannote committed:
added Tower and deleted Forests

titannote committed:
Sand Texture under River

titannote committed:
Small Forest

Proidott committed:
trees

height checked some trees to make the forrest area look more realistic unfortunately i wasn't able to check if a titan could fit through because my computer was being buggy

Proidott committed:
river and trees

started working on the Forrest

Proidott committed:
trees

worked on trees and started new path

Proidott committed:
more trees

Velsh-arc committed:
Barriers added

dark3art committed:
Merge branch 'development' into #108-Standoff

Proidott committed:
trees

finished out lining where the tree area will be

Proidott committed:
new colliders

dark3art committed:
Merge branch 'development' into #108-Standoff

dark3art committed:
deleted all usseles files, added in the prefab files getting ready to fix it all

dark3art committed:
fixed some painting

selahgreen committed:
I added a small village on both sides of stand-off

dark3art committed:
put down forest fixed the outside the boundries and added titan spawns

selahgreen committed:
added trees to the town and fixed models

dark3art committed:
fixed some more out of boundries

dark3art committed:
fixed some minor issues and made a build
